doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -166,6 +166,17 @@ Gazell samples
 
     There are now four different sample pages, where each Host sample must be used along with its corresponding Device sample.
 
+Zigbee samples
+--------------
+
+* Added support of factory reset functionality from :ref:`lib_zigbee_application_utilities` in the following:
+
+  * :ref:`zigbee_light_bulb_sample`
+  * :ref:`zigbee_light_switch_sample`
+  * :ref:`zigbee_network_coordinator_sample`
+  * :ref:`zigbee_shell_sample`
+  * :ref:`zigbee_template_sample`
```

Comment:
+ Zigbee weather station?

Reply:
@tomchy There is separate entry for ZWS as a newly added app, so there is no point in stating that it was also modified. There is a distinction between apps and samples